style.css

```diff
@@ -0,0 +1,386 @@
+.menu {
+  display: flex;
+  justify-content: right;
+  padding: 20px;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+  margin-right: 24px;
+  margin-left: -24px;
+}
+
+.greetings {
+  justify-content: center;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+}
+
+.job {
+  color: #36b37e;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+}
+
+.socials {
+  display: flex;
+  gap: 1rem;
+  justify-content: center;
+  list-style-type: none;
+  margin-top: 14px;
+  padding-bottom: 2px;
+}
+
+.button {
+  color: #172b4d;
+  justify-content: center;
+  display: flex;
+  padding-bottom: 10px;
+}
+
+.headline {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 46px;
+  text-align: center;
+  overflow-y: auto;
+}
+
+.section-container9 {
+  background-image: url("images/illustrator-mobile.svg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+}
+
+.background-image {
+  width: 100%;
+  position: absolute;
+}
+
+.line-break::after {
+  content: '';
+  width: 48px;
+  height: 4px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 24px;
+  background: #36b37e;
+  border-radius: 24px;
+  display: flex;
+  justify-content: center;
+}
+
+.works {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  padding-top: 40px;
+}
+
+.section-placeholder {
+  margin-top: 50px;
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 58px;
+  margin-right: 24px;
+  margin-left: 24px;
+  align-items: center;
+  padding: 64px 24px 64px 24px;
+}
```

Comment:
- [ ] This is a close one but it seems like the width of the project cards is a bit narrower when compared with the Figma design as shown below and this is mainly because of the spacing from the left and right, so I kindly suggest you reduce the horizontal spacing (margin & padding) so the project card is a bit wider.
- Your design:
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/104968236/7ab1fe30-d0ec-4303-ba80-e0b00eb2f632)
- Figma design:
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/104968236/4fbe00e5-cf12-4acc-841d-4832802d6b35)